test/TranscodeJobs.js

```diff
@@ -0,0 +1,79 @@
+var TranscodeJobs = artifacts.require("./TranscodeJobs.sol");
+
+import abi from "ethereumjs-abi";
+import BigNumber from "bignumber.js";
+
+contract('TranscodeJobs', function(accounts) {
+    let transcodeJobs;
+
+    before(async function() {
+        transcodeJobs = await TranscodeJobs.new();
+    });
+
+    describe("shouldVerifySegment", function() {
+        it("should return true for a segment eligible for verification", async function() {
+            let segNum = 0;
+            const blockNum = web3.eth.blockNumber;
+            const block = await web3.eth.getBlock(blockNum);
+            const blockHash = block.hash;
+            const verificationRate = 4;
+
+            // Find an eligible segment
+            let hash = abi.soliditySHA3(["uint256", "bytes", "uint256"], [blockNum, Buffer.from(blockHash.slice(2), "hex"), segNum]).toString("hex");
+            let hashNum = new BigNumber(hash, 16);
+            let res = hashNum.mod(verificationRate);
+
+            while (res != 0) {
```

Comment:
Let's add a comment about why we're iterating here to find a suitable candidate segment number.